src/Bicep.Core.UnitTests/Diagnostics/LinterRuleTests/InterpolateNotConcatRuleTests.cs

```diff
@@ -373,8 +373,17 @@ var b
         )]
         [DataRow(
```

Comment:
Fyi, there's at least one other testcase with one arg, so it should now be failing.

Reply:
found and fixed